android/javatests/org/chromium/chrome/browser/BytecodeTest.java

```diff
@@ -507,6 +515,9 @@ public void testMethodsExist() throws Exception {
         Assert.assertTrue(
                 methodExists("org/chromium/chrome/browser/preferences/ChromePreferenceKeyChecker",
                         "getInstance", false, null));
+        Assert.assertTrue(
```

Comment:
@sujitacharya2005 we don't need this test as we only redirect constructor.

Reply:
and since we override the function, if we want to check if the function is avaiable in the base class, you need to include `*** onItemSelected(...);` in `apk_for_tests.flags`